tests/it/main.rs

```diff
@@ -470,6 +470,16 @@ fn formatting() {
     cmd!("cargo fmt --all -- --check").run().unwrap()
 }
 
+#[test]
+#[allow(unused_must_use)]
+fn string_escapes() {
+    setup();
+
+    cmd!("\"hello\"");
+    cmd!("\"\"\" asdf\" \"");
+    cmd!("\\\\\\\\\\\\\\\\\\\\\\\\\\");
```

Comment:
Let's add some asserts here, I think the best would be to use `cmd.to_string()` to check this:

https://github.com/matklad/xshell/blob/69ce10674ad73380eef6a4b6f0797343c476296d/tests/it/main.rs#L81-L93